docs/multiple-connections.md

```diff
@@ -291,6 +292,20 @@ try {
         
 ```
 
+If you want to use `slave` in raw queries, you also need to explicitly specify the query runner.
+
+```typescript
+
+const slaveQueryRunner = connection.createQueryRunner("slave");
+await connection.query('SELECT * FROM users WHERE id = $1', [userId], slaveQueryRunner)
+  .then((result) => {
+    // Do your stuff
+  })
+  .finally(() => {
+    return slaveQueryRunner.release()
+  })
```

Comment:
If this is going to be a separate query please match the `try`/`finally` syntax as above.